test/Tizen.NUI.Tests/Tizen.NUI.Devel.Tests/testcase/public/Images/TSPixelData.cs

```diff
@@ -76,31 +76,6 @@ public void PixelDataConstructorByIntPtr()
             tlog.Debug(tag, $"PixelDataConstructorByIntPtr END (OK)");
         }
 
-        [Test]
-        [Category("P1")]
-        [Description("PixelData Url.")]
-        [Property("SPEC", "Tizen.NUI.PixelData.Url A")]
-        [Property("SPEC_URL", "-")]
-        [Property("CRITERIA", "PRW")]
-        [Property("AUTHOR", "guowei.wang@samsung.com")]
-        public void PixelDataUrl()
-        {
-            tlog.Debug(tag, $"PixelDataUrl START");
-
-            byte[] buffer = new byte[1024];
-
-            var testingTarget = new PixelData(buffer, 1024, 100, 150, PixelFormat.L8, PixelData.ReleaseFunction.Free);
-            Assert.IsNotNull(testingTarget, "Can't create success object PixelData");
-            Assert.IsInstanceOf<PixelData>(testingTarget, "Should be an instance of PixelData type.");
-
-            var result = testingTarget.Url;
```

Comment:
Could you just change it as `GenerateUrl()`;

Reply:
Done